src/libexpr/primops.cc

```diff
@@ -801,8 +790,10 @@ static void prim_addErrorContext(EvalState & state, const PosIdx pos, Value * *
         v = *args[1];
     } catch (Error & e) {
         PathSet context;
-        e.addTrace(nullptr, state.coerceToString(pos, *args[0], context,
-                "while evaluating the error message passed to builtins.addErrorContext").toOwned());
+        auto message = state.coerceToString(pos, *args[0], context,
+                "while evaluating the error message passed to builtins.addErrorContext",
+                false, false).toOwned();
+        e.addTrace(nullptr, message);
```

Comment:
@tomberek: here for example: state.coerceToString(pos, *args[0], context, "while evaluating the error message passed to builtins.addErrorContext"). The string ("while ...") was passed in place of the first bool argument (copyToStore) while it should really be passed after three bools corresponding to the default values of the three parameters.